### DISCUSSION TITLE
Enderchest Permissions

### DISCUSSION THREAD

**Discussion by TheExceed** [Zrips/CMI#4614] [Feature Requests]
Extend the Enderchest to have permission based row counts, up to 6 rows.

Example:
cmi.enderchest.# - Number being how many rows they would be able to use.
cmi.enderchest.3 - would allow 3 rows to the enderchest

**Why?**
Well, CMI, to me, is an all in one solution for most basic necessities in minecraft and this would be a very nice way to give players a bonus by increasing their enderchest size. Sure, there are loads of other enderchest plugins but I think this would just continue to add to the value of your plugin.

**Comment by CloudeLecaw** [comment]
cmi uses playerfile for inventories and enderchet. the normal player enchest is limited to 3 slots. So you cant have 6 rows.
only 1 2 or 3

**Comment by TheExceed** [comment]
I have seen it done before while using playerfiles though. Unless the Dev lied. But I doubt they had the reason too as it was free. But anywho, if he can figure it out, that would be great.
